types/node/v12/tslint.json

```diff
@@ -3,6 +3,7 @@
   "rules": {
     "ban-types": false,
     "no-empty-interface": false,
+    "no-outside-dependencies": false,
```

Comment:
What's this for?

Reply:
The (GitHub Actions) CI [was failing](https://github.com/DefinitelyTyped/DefinitelyTyped/runs/1359414000#step:5:3131) without it. It's unrelated to this PR, the bot just won't cooperate unless the CI is passing.